### ISSUE TITLE
Feature Request : Adding export to image and/or pdf 

### ISSUE THREAD

**Issue by Miatec** [dartungar/obsidian-mermaid#49]
Hello,

First of all, thank you for this work, I don't use mermaid very often in obsidian but It's a pleasure to use when needed!

I'm summiting this feature request to your good will because sometimes I want to share my graph in my notes by mail. Right now the only solution embedded is to screenshot the graphs and the result is not always perfect, in particular with big graphs : either I need some editing to paste several screenshots or the labels are unreadable.

I've come to the solution to copy my mermaid code in https://mermaid.live/ and use its export options :

![image](https://github.com/dartungar/obsidian-mermaid/assets/12949689/c0b2b1ef-f233-48d3-a7ad-cd7c1319b792)

I guess the export option should be accessible from  editing view, adding a icon nex to edit button, like this : 

![export_obsidian_mermaid](https://github.com/dartungar/obsidian-mermaid/assets/12949689/b9dd4b7e-b1cd-4a59-8f7c-c5a396e0c54c)

TL;DR:
Could you please add : 
- A PNG export, with customizable size
- A SVG export

Thank you for your time :-)

**Comment by dartungar** [comment]
@Miatec hey there! This would be neat, but not trivial, and I don't have much time right now. 
Adding "help wanted" tag for anyone who'd want to help!

**Comment by dartungar** [comment]
@Miatec also, you can export the whole file in PDF via Obsidian.
